docs/source/WritingTests.rst

```diff
@@ -827,15 +827,38 @@ a timeout of 3 seconds before Avocado ends the test forcefully by sending a
 :class:`avocado.core.exceptions.TestTimeoutError`.
 
 
-Test Tags
-=========
+Docstring Directives
+====================
+
+Some Avocado features, usually only available to instrumented tests,
+depend on setting directives on the test's class docstring.  The
+standard prefix used is ``:avocado:`` followed by the directive
+itself, such as ``:avocado: directive``.
+
+This is similar to docstring directives such as ``:param my_param:
```

Comment:
nitpick: Docstring uses `@param my_param: ...`, it's sphinx who changed it to `:param my_param: ...`...